rocks-lib/src/tree/list.rs

```diff
@@ -22,30 +22,32 @@ impl Tree {
                     .unwrap()
                     .split_once('@')
                     .unwrap();
-                (name.to_string(), version.to_string())
+                (
+                    PackageName::new(name.to_string()),
+                    PackageVersion::parse(version)
+                        .expect("invalid version found in rocktree. This is a bug!"),
```

Comment:
suggestion: include the path to the error message

Reply:
All of these errors are currently placeholders, either serving as a "please implement a serious error handling strategy here in the future" or "issue a nice `ariadne` error here so the user isn't confused".

I'm banking on the fact that since we'll have a lot of tests we can aggressively refactor for better error messages and less `clone()` invocations without fearing any breakages once the project hits its MVP stage. I'll add the path here for clarity though :)